content/tutorials/set-count.md

```diff
@@ -0,0 +1,98 @@
+---
+authors:
+  - name: MaSera
+    title: Contributor
+    url: https://github.com/Maseraphina
+    image_url: https://github.com/Maseraphina.png
+tags: [set count, multiple counting channels]
+---
+import {
+  DiscordActionRow as ActionRow,
+  DiscordAttachments as Attachments,
+  DiscordBold as Bold,
+  DiscordButton as Button,
+  DiscordCommand as Command,
+  DiscordEmbedDescription as Description,
+  DiscordMessages as Discord,
+  DiscordEmbed as Embed,
+  DiscordMessage as Message,
+  DiscordMention as Mention,
+} from "@skyra/discord-components-react";
+
+# How do I change the count?
+This tutorial will give you more information on how to change a count in your server.<br/>(regardless of whether you have 1 or more counting channels in your server)
+
+<!--truncate-->
+#### First of all, why is my count sometimes wrong?
+
+The count can be incorrectdue to various reasons:
+1. You or someone else deleted the previous count, so it looks messed up.
+2. Countr has been offline
+3. It bugged out due to high latency.
+  <br/>or
+4. you've just invited countr in your server and there already  is a counting channel.
```

Comment:
- remove the `<br/>` and put the `or` on the same line as the fourth one
- capitalize the start of the sentence, or make all bullet/number points lowercased first
- remove extra space